public/app/features/canvas/elements/button.tsx

```diff
@@ -45,22 +52,27 @@ export const buttonItem: CanvasElementItem<ButtonConfig, ButtonData> = {
   display: ButtonDisplay,
 
   defaultSize: {
-    width: 32,
+    width: 78,
     height: 32,
   },
 
   getNewOptions: (options) => ({
     ...options,
+    config: {
+      text: {
+        mode: TextDimensionMode.Fixed,
+        fixed: 'Button',
+      },
+      api: defaultApiConfig,
+    },
     background: {
       color: {
         fixed: 'transparent',
       },
     },
     placement: {
-      width: 32,
```

Comment:
Is removing these width and height attributes intentional?

Reply:
yes - they were not working properly and we have a task to fix the placement issues, so we'll update there with the right values.